frontend/public/locales/zh/public.json

```diff
@@ -1238,7 +1243,6 @@
   "Exceptions": "例外",
   "with exceptions": "带有例外",
   "Any port": "任何端口",
-  "NetworkPolicy details": "网络策略详情",
```

Comment:
This translation is missing in 4.17 on NetworkPolicy details -> Details Tab

Reply:
> This translation is missing in 4.17 on NetworkPolicy details -> Details Tab
The NetworkPolicy details  code is not console